entrypoint.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 python3 -m flask db upgrade
-gunicorn --worker-tmp-dir /dev/shm -w 2 --worker-class=gthread --threads 2 -b 0.0.0.0:5666 --log-file /app/aguni.log --log-level info "arcsi:create_app('../config.py')"
+gunicorn --worker-tmp-dir /dev/shm -w 2 --worker-class=gthread --threads 2 -b 0.0.0.0:5666 --log-file /app/aguni.log --log-level debug "arcsi:create_app('../config.py')"
```

Comment:
We shouldn't commit `debug` level, or?

Reply:
Actually, I thought yes. The log isn't too verbose even in debug mode (i.e., I see no disk concerns) but they can help us understand tricky cases during production (sharp operation). I see most debug log information we have now as "verbose" rather than "development/debug". What do you think?

Reply:
I see. In that case I might want to rename most of the current debug level logs to info and still change this back. Just the notion in my mind running anything on debug on prod is hard to swallow. But this is mostly aesthetics so we can go ahead w/ this

Reply:
I agree. One thing which is not fully cosmetical is the built in gunicorn debug log entries, which are very informative during normal operation, in my opinion (e.g., "Closed connection" every time a route request is served). What I mean is that if we change the above log entries to info mode, and we switch to info mode on the prod server, then these default gunicorn entries would disappear.